snap/snapcraft.yaml

```diff
@@ -124,10 +116,13 @@ parts:
 
       CPU_COUNT=$(lscpu --parse=CPU | grep -v "^#" | wc -l)
 
+      export DESTDIR="$(readlink --canonicalize ${SNAPCRAFT_PART_INSTALL})"
+      export PREFIX=/usr
```

Comment:
Is this Makefile really accepting `PREFIX` as an environment variable instead of Makefile veriable? In any case, if this is the installation prefix it should be `$SNAPCRAFT_PART_INSTALL/usr` instead of  `/usr`, otherwise files will be installed on the host's system directories.

Reply:
Yes, the `Makefile` uses
```make
DESTDIR ?=
PREFIX ?= /usr/local
```